File: C++/Manual 09/Question03.cpp
#include <iostream>

using namespace std;

struct Circle
{
    double radius;
    double diameter;
    double area;
    static constexpr double pi = 3.1415; // Using static constexpr for constant values

    void Input()
    {
        cout << "Enter Diameter: ";
        cin >> diameter;
        radius = diameter / 2;
        area = radius * radius * pi;
        cout << "Radius is: " << radius << " units\n";
        cout << "Area is: " << area << " square units\n";
    }
};

int main()
{
    Circle value;
    value.Input();
    return 0;
}

File: Python/Manual 09/Question03.py
class Circle:
    def __init__(self, diameter):
        self.diameter = diameter

    def printArea(self):
        radius = self.diameter / 2
        area = 3.14 * (radius**2)
        print(f"Area of the circle: {area}")


def main():
    myCircle = Circle(10)
    myCircle.printArea()


main()
